gridded_etl_tools/utils/store.py

```diff
@@ -233,14 +233,20 @@ def fs(self, refresh: bool = False) -> s3fs.S3FileSystem:
         ----------
         refresh : bool
             If set to `True`, a new `s3fs.S3FileSystem` will be created even if this object has one already
+        profile : str
+            Manually specify the name of the AWS profile to be used when creating an S3FileSystem.
+            Overrides the default profile grabbed silently by aiobotocore.
 
         Returns
         -------
         s3fs.S3FileSystem
             A filesystem object for interfacing with S3
         """
         if refresh or not hasattr(self, "_fs"):
-            self._fs = s3fs.S3FileSystem()
+            if profile:
+                self._fs = s3fs.S3FileSystem(profile=profile)
```

Comment:
What would happen if we passed `profile=None` here? Do we need the if/else?

Reply:
You’re right, this would simplify things

Reply:
Zapped!